documentation/modules/collected-logs-cr-info.adoc

```diff
@@ -0,0 +1,70 @@
+// Module included in the following assemblies:
+//
+// * documentation/doc-Migration_Toolkit_for_Virtualization/master.adoc
+
+[id="collected-logs-cr-info_{context}"]
+= Collected logs and custom resource information
+
+You can download logs and custom resource (CR) `yaml` files for the following targets by using the {project-short} web console or the command line interface (CLI):
+
+* Migration plan: Web console or CLI.
+* Virtual machine: Web console or CLI.
+* Namespace: CLI only.
++
+[NOTE]
+====
+If you specify a non-existent target, the `must-gather` archive file is empty.
+
```

Comment:
No need for this note here. It is relevant only for CLI and the note in CLI section is good